Sources/MapboxMaps/Style/SupportedLanguage.swift

```diff
@@ -0,0 +1,45 @@
+/// A list of languages that our maps support for localization
+public enum SupportedLanguage: String, CaseIterable {
+
+    /// Support for Arabic (if available)
+    case arabic = "ar"
+
+    /// Support for Chinese (if available)
+    case chinese = "zh"
+
+    /// Support for English (if available)
+    case english = "en"
+
+    /// Support for French (if available)
+    case french = "fr"
+
+    /// Support for German (if available)
+    case german = "de"
+
+    /// Support for Italian (if available)
+    case italian = "it"
+
+    /// Support for Japanese (if available)
+    case japanese = "ja"
+
+    /// Support for Korean (if available)
+    case korean = "ko"
+
+    /// Support for Portuguese (if available)
+    case portuguese = "pt"
+
+    /// Support for Russian (if available)
+    case russian = "ru"
+
+    /// Support for Simplified Chinese (if available)
+    case simplifiedChinese = "zh-Hans"
+
+    /// Support for Spanish (if available)
+    case spanish = "es"
+
+    /// Support for Traditional Chinese (if available)
```

Comment:
Can we make a note that this is not supported in mapbox.mapbox-streets-v7?

Reply:
sure